source/components/molecules/CaseCard/CaseCard.tsx

```diff
@@ -37,6 +37,7 @@ interface CaseCardProps {
   showBookingDate?: boolean;
   bookingDate?: string;
   bookingTime?: string;
+  smallDateTimeCard?: boolean;
```

Comment:
how about changing the property to `dateTimeCardSize` which is a string of with value "small" or "large" ?